addons/account/static/src/js/section_and_note_fields_backend.js

```diff
@@ -68,9 +68,10 @@ var SectionAndNoteListRenderer = ListRenderer.extend({
      * @private
      */
     _renderView: function () {
-        var def = this._super();
-        this.$el.find('> table').addClass('o_section_and_note_list_view');
-        return def;
+        var self = this;
+        return this._super().then(function() {
```

Comment:
function ()

Reply:
also, i always try to use this._super.apply(this, arguments), in case the parent API would change